server/service/src/processors/general_processor.rs

```diff
@@ -11,15 +11,15 @@ use crate::{
     service_provider::{ServiceContext, ServiceProvider},
 };
 
-use super::contact_form::QueueContactEmailProcessor;
+use super::{contact_form::QueueContactEmailProcessor, load_plugin::LoadPlugin};
 
 #[derive(Error, Debug)]
 pub(crate) enum ProcessorError {
-    #[error("{0:?} not found: {1:?}")]
+    #[error("{0} not found: {1}")]
```

Comment:
Chnaged error mapping, tehnically should just do `[source]` or from for existing error and they will be added to the error message without needing to print them out